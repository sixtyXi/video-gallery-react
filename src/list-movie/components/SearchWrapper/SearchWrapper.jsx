import React from 'react';

import styles from './SearchWrapper.scss';
import SearchBtnContainer from '../../../containers/SearchBtnContainer/SearchBtnContainer';
import SearchFieldContainer from '../SearchFieldContainer/SearchFieldContainer';
import SearchFilterContainer from '../SearchFilterContainer/SearchFilterContainer';

export const SearchWrapper = () => {
  return (
    <form className={styles.searchWrapper}>
      <h1 className={styles.pageTitle}>Find your movie</h1>

      <SearchFieldContainer />

      <SearchFilterContainer />

      <SearchBtnContainer />
    </form>
  );
};
