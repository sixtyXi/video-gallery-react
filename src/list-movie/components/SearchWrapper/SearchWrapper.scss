@import '../../../shared/_colors.scss';
@import '../../../shared/_typography.scss';

.searchWrapper {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  padding: 20px 0;
  color: $text-light;
  font-variant: all-small-caps;
}

.pageTitle {
  padding: 0;
  text-transform: uppercase;
  font-size: $size-l;
}
