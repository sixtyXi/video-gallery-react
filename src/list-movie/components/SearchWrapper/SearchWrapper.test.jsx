import React from 'react';
import { shallow } from 'enzyme';

import { SearchWrapper } from './SearchWrapper';

describe('SearchWrapper', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<SearchWrapper />);

    expect(wrapper).toMatchSnapshot();
  });
});
