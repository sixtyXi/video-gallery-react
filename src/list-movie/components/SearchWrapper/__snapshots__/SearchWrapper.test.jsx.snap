// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchWrapper renders correctly 1`] = `
<form
  className="searchWrapper"
>
  <h1
    className="pageTitle"
  >
    Find your movie
  </h1>
  <SearchFieldContainer />
  <SortContainer
    defaultFilterIdx={0}
    filters={
      Array [
        Object {
          "title": "title",
          "value": "title",
        },
        Object {
          "title": "genre",
          "value": "genre",
        },
      ]
    }
    title="Search by"
    type="btn"
  />
  <SearchBtn
    className="searchBtnPrimary"
  />
</form>
`;
